src/plugins/links/public/editor/open_editor_flyout.tsx

```diff
@@ -55,6 +57,14 @@ export async function openEditorFlyout(
   }
 
   return new Promise((resolve, reject) => {
+    const closeEditorFlyout = (editorFlyout: OverlayRef) => {
+      if (overlayTracker) {
+        overlayTracker.clearOverlays();
+      } else {
+        editorFlyout.close();
+      }
```

Comment:
We can no longer assume that the `overlayTracker` (from the Dashboard app) exists, since this flyout can now be opened outside of the Dashboard app - therefore, if the `overlayTracker` does not exist, we need to rely on the normal flyout `close` method.